server/models/Request.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    userId: { type: mongoose.ObjectId, required: true, ref: "User" },
```

Comment:
Just to clarify my previous comment about this, I think `ownerId` might be a better name for this field since `user` is a bit ambiguous.